src/views/LoginSuccess.vue

```diff
@@ -112,10 +117,24 @@ export default class LoginSuccess extends Vue {
         this.done();
     }
 
-    private done() {
+    private async done() {
         if (!this.walletInfos.length) throw new Error('WalletInfo not ready.');
 
-        this.result = this.walletInfos.map((walletInfo) => ({
+        if (BrowserDetection.isIOS() || BrowserDetection.isSafari()) {
```

Comment:
Could be done in `mounted()` after the `WalletInfoCollector` did its job, but before the Accounts are added to the indexedDB.